cmd/wallet-sdk-gomobile/credentialquery/credentialquery.go

```diff
@@ -44,7 +55,7 @@ func NewQuery(documentLoader api.LDDocumentLoader) *Query {
 }
 
 // Query returns credentials that match PresentationDefinition.
-func (c *Query) Query(query, contents []byte) ([]byte, error) {
+func (c *Query) Query(query []byte, contents Credentials) ([]byte, error) { //nolint: gocyclo
```

Comment:
The `Credentials` argument needs to be a pointer, otherwise `gomobile` will skip generating bindings for this method